src/style.css

```diff
@@ -0,0 +1,18 @@
+body {
+  margin: 0;
+}
+
+#square1 {
```

Comment:
use names that describe the content or its behavior
numbers don't explain much about your page